### PR TITLE
Website version 1.0?

### PR DESCRIPTION
**Pull request by calebn**
I think this PR represents a shippable version, there is still lots to be done but we have the minimum amount of information to move past the coming soon beta stage.

- Updated homepage, no longer coming soon, now has some real info
- Added related assets
- Minor code formatting/refactoring

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,31 +1,155 @@
...
+import SectionContainer from '../components/styles/SectionContainer.styled';
+import Image from 'next/image';
+import SoCBanner from '../images/seed-of-cain/banner.jpg';
+import SoCLogo from '../images/seed-of-cain/logo.png';
+import {
+	RobotIconHexagon,
+	BlockchainIconHexagon,
+	MicrochipIconHexagon,
+} from '../components/HexagonIcons';
+import LogoWithText from '../components/LogoWithText';
```
Comment (tr3mulant) [inline pages/index.js:17]: is this supposed to be `LogoTextImage`?

File: pages/index.js [hunk 2]
```diff
@@ -1,31 +1,155 @@
...
-				<ComingSoon>
-					<ComingSoonImage />
-					{/* <SignUp>Sign up to receive email updates</SignUp>
-					<NewsletterSubscribe /> */}
-				</ComingSoon>
+				<StyledSection>
+					<SectionContainer>
+						<PsychoDescriptionContainer>
+							<PsychoDescriptionImage>
+								<LogoWithText />
```
Comment (tr3mulant) [inline pages/index.js:157]: `LogoTextImage`?

File: components/ComingSoonImage.js [hunk 3]
```diff
@@ -2,12 +2,12 @@ import styled, { useTheme } from 'styled-components';
 import MotionLogo from './styles/MotionLogo.styled';
 
 const ImageContainer = styled.div`
-	margin-bottom: ${(props) => props.theme.spaces.xxl};
+	margin-bottom: ${({ theme }) => theme.spaces.xxl};
```
Comment (tr3mulant) [inline components/ComingSoonImage.js:5]: Nice move here.

**Comment by tr3mulant** [comment]
Fetched your pr, built and I couldn't due to missing `LogoWithText`. I swapped it out for `LogoTextImage` and it built just fine. 

Dude, that home page is sick!

What do you think about changing the aspect ratio for small devices for the SoC banner. Idea is that there is more image on mobile if the height > width. Perhaps we use the original poster image in the tall orientation? I think it is `images/seed-of-cain/pillar.jpg`

I only say this because the art is awesome and I want to give it more real estate because the desktop version I was blown away.

File: pages/index.js (hunk 1 above)
Comment (calebn) [reply pages/index.js:17]: I forgot to include something in my commit, I'll get it added.

File: pages/index.js (hunk 2 above)
Comment (calebn) [reply pages/index.js:157]: > What do you think about changing the aspect ratio for small devices for the SoC banner. Idea is that there is more image on mobile if the height > width. Perhaps we use the original poster image in the tall orientation? I think it is `images/seed-of-cain/pillar.jpg`

I like this idea, I'll make an update.

**Comment by calebn** [comment]
@tr3mulant I'm going to go ahead and merge this now, thanks for the review and catching those missing files.

**Pull request merged**